benchmark/src/jmh/kotlin/kotlinx/benchmarks/json/TwitterFeedStreamBenchmark.kt

```diff
@@ -88,4 +115,24 @@ open class TwitterFeedStreamBenchmark {
             objectMapper.readValue(it, MicroTwitterFeed::class.java)
         }
     }
+
+    @Benchmark
+    fun decodeMicroTwitterOkioStream(): MicroTwitterFeed {
+        // It seems there is no optimal way to reuse `bytes` between benchmark, so we are forced
+        // to write them to buffer every time.
+        // Note that it makes comparison with Jackson and InputStream integration much less meaningful.
+        val b = OkioBuffer()
+        b.write(bytes)
+        return jsonIgnoreUnknwn.decodeFromBufferedSource(MicroTwitterFeed.serializer(), b)
+    }
+
+    @Benchmark
+    fun decodeMicroTwitterKotlinxIoStream(): MicroTwitterFeed {
```

Comment:
For some reason, this particular benchmark shows worse results compared to Okio version. It doesn't block integration in any way, I'll check the root cause later.

Reply:
That's a perf issue on the kotlinx-io size: an extension function to read code point values is defined only on `Source` and it performs a few actions to fill the inner buffer before decoding the code point. In Okio, that's a member function with a different (and optimized) implementation for `Buffer`. 
kx-io should do the same: https://github.com/Kotlin/kotlinx-io/issues/342.